truss/templates/server.Dockerfile.jinja

```diff
@@ -89,7 +91,18 @@ COPY ./{{ config.model_module_dir }} /app/model
 
 
 {% block run %}
-    {%- if config.live_reload %}
+    {%- if config.docker_server %}
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+        curl nginx python3-pip && \
+        rm -rf /var/lib/apt/lists/*
+RUN pip3 install supervisor==4.2.5
+COPY ./proxy.conf /etc/nginx/conf.d/proxy.conf
+RUN mkdir -p /var/log/supervisor
+COPY supervisord.conf /etc/supervisor/supervisord.conf
+ENV SUPERVISOR_SERVER_URL http://localhost:8080
```

Comment:
nit: consider moving some of these file paths / urls into template variables, so constants can be centralized in one place